dessia_common/exports.py

```diff
@@ -50,8 +51,9 @@ def __init__(self, selector: Optional[str], extension: str, method_name: str, te
         self.args = args
 
     def to_dict(self):
-        return {"selector": self.selector, "extension": self.extension, "method_name": self.method_name,
-                "text": self.text, "export_name": self.export_name, "args": self.args}
+        return {"selector": self.selector, "extension": self.extension,
```

Comment:
Do you still use the 80 char max line length rule ? We decided to set it to 120 now

Reply:
Yes I do.